packages/formatting/src/utils/detection.ts

```
@@ -0,0 +1,198 @@
+    };
+  }
+}
+
+/**
+ * Detect all available formatters
+ */
+export async function detectAvailableFormatters(): Promise<
+  Result<FormatterDetectionResult, Error>
+> {
+  try {
+    const formatters: FormatterType[] = ['prettier', 'biome', 'remark'];
```

ESLint

[error] Array type using 'FormatterType[]' is forbidden. Use 'Array<FormatterType>' instead.

(@typescript-eslint/array-type)